docs/DeveloperGuide.md

```diff
@@ -1118,6 +1122,4 @@ testers are expected to do more *exploratory* testing.
         Expected: CONNECTIONS launches successfully with 0 contacts. 
        
     1. Other incorrect formatting of data or invalid fields: birthday to `00000000`, phone to `mynumber`, `...`
-        Expected: Similar to previous
-
-1. _{ more test cases …​ }_
+        Expected: CONNECTIONS launches successfully with 0 contacts.
```

Comment:
Same here, should leave one line in between or use \<br>, if not looks like this
<img width="1141" alt="Screenshot 2021-11-08 at 12 53 49 AM" src="https://user-images.githubusercontent.com/69960220/140654217-e07eaf61-0c70-4411-a41b-b965735a1bed.png">